File: 8KYU/Calculate BMI/solution.c
/*    N.B. do not allocate memory,
   instead: return a string literal  */

char *bmi(int weight, double height) {

  double bmi = (weight / (height * height));
  if (bmi <= 18.5)
    return "Underweight";
  if (bmi <= 25.0)
    return "Normal";
  if (bmi <= 30)
    return "Overweight";
  
  return "Obese";
}

File: 8KYU/Calculate BMI/solution.rb
def bmi(weight, height)
    bmi = (weight / (height * height))
    case bmi
      when 0..18.5
        "Underweight"
      when 18.5..25
        "Normal"
      when 25..30
        "Overweight"
      else
        "Obese"
      end
end